### PR TITLE
Add Damageable Object Basics Tutorial

### PR DESCRIPTION
**Pull request by CommanderFoo**

### CODE REVIEW

File: src/tutorials/damageable_object_basics.md [hunk 1]
```diff
@@ -0,0 +1,750 @@
...
+
+#### Destroy on Death Client Template
+
+The **Destroy on Death Client Template** will spawn a template on the client at the location of the damageable object after the **Destroy on Death Delay**. This is an efficient way of spawning visual effects, and anything else for the client. For example, when a damageable object is destroyed, smoke effect is spawned.
+
+In the video below, you will see a template is set for the property that will be spawned when the damageable object is destroyed.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/DamageableObjects/Basics/client_template_property.mp4" type="video/mp4" />
```
Comment (DoubleA-MG) [inline src/tutorials/damageable_object_basics.md:144]: Hmm... this is weird.

In the preview the Destroy on Death Client Template video and Destroy on Death Networked Template video are switched. I think you may have accidently named the videos incorrectly.

File: src/tutorials/damageable_object_basics.md [hunk 2]
```diff
@@ -0,0 +1,750 @@
...
+
+The **Destroy on Death Networked Template** will spawn a networked template at the location of the damageable object after the **Destroy on Death Delay**. This is useful for spawning other gameplay objects from a damageable object that is destroyed. For example, when a damage object is destroy, it could spawn a pickup for all players to collect.
+
+In the video people, you will see a template is set for the property that will be spawned when the damageable object is destroyed.
+
+!!! tip "Make sure that the template you use for this property is **Networked**."
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/DamageableObjects/Basics/networked_template_property.mp4" type="video/mp4" />
```
Comment (DoubleA-MG) [inline src/tutorials/damageable_object_basics.md:158]: This is the other video that was misnamed mentioned before.

File: src/tutorials/damageable_object_basics.md [hunk 3]
```diff
@@ -0,0 +1,750 @@
...
+In this section you will setup the most basic damageable object that will have some geometry and collision that can be destroyed, in future sections you will be adding additional layers to improve the dummy.
+
+For the basic dummy, the properties on the damageable object will not be changed, which will show you how easy it can be to create objects that can be destroyed by the player.
+
+!!! tip "Destroying Damageable Objects"
+    To destroy any of the damageable objects throughout this tutorial, you will need to add a weapon from **Core Content**. For example, search for `advanced assault` to find the **Advanced Assault Rifle** and add it to your **Hierarchy** so you can pick it up when you play the game.
+
+### Create Damageable Object
+
+When you want to create your own custom damageable objects, you will want to start with the basic **Damageable Object** from **Core Content**. On its own, it is not that useful because it has no visual, and no collision. All damageable objects need some collision for it to register damage, and in most cases you will want some geometry so you can see the what to damage.
```
Comment (DoubleA-MG) [inline src/tutorials/damageable_object_basics.md]: reword that final sentence. *so you can see ~the~ what to damage.

File: src/tutorials/damageable_object_basics.md [hunk 4]
```diff
@@ -0,0 +1,750 @@
...
+1. In **Core Content** search for `Damageable Object` and add it to your **Hierarchy**.
+2. Rename **Damageable Object** to `Exploding Damageable Object`.
+3. In **Project Content** search for `Exploding Dummy` and add it as a child of **Exploding Damageable Object**.
+4. Move the **Exploding Damageable Object** so it is not in the ground, and away from the player.
+
+![!Exploding Dummy](../img/DamageableObjects/Basics/exploding_dummy.png){: .center loading="lazy" }
+
+### Add Destroy on Death Networked Template
+
+When the **Exploding Damageable Object** is killed, a template will be spawned that contains a Lua script that will loop over all the meshes that support **Debris Physics** by setting `isSimulatingDebrisPhysics` to `true`. The template that will be spawned has already been setup so that objects which support **Debris Physics** are in a **Supported** group.
```
Comment (DoubleA-MG) [inline src/tutorials/damageable_object_basics.md]: link the debris physics reference: ../references/debris_physics/

**Review by DoubleA-MG — CHANGES_REQUESTED** [review]
I like the lesson. To the point and avoided most of the tedious work to focus on Damageable Objects and not scripting.

File: src/tutorials/damageable_object_basics.md (hunk 1 above)
Comment (CommanderFoo) [reply src/tutorials/damageable_object_basics.md:144]: Well spotted. Not sure why I done that lol

**Review by DoubleA-MG — APPROVED** [review]

**Pull request merged**